omaha/net/cup_ecdsa_pubkey.1.h

```diff
@@ -19,13 +19,13 @@
 //     * Uncompressed header byte (0x04)
 //     * Gx coordinate (256-bit integer, big-endian)
 //     * Gy coordinate (256-bit integer, big-endian)
-{0x0a,
```

Comment:
This is public key generated by myself used to check authenticity of update server. Private key will be used to check it on server side.

Reply:
should we move the public key as a secret in github action in the future? what do you think?
if yes will create a task

Reply:
We will do that in future